Expensify/App
# Improve inline icons - Address 5873

### Details
Adds the `inline` prop to `<Icon />` fixing vertical alignment issues when icons are inside `<Text />`

### Fixed Issues

$ https://github.com/Expensify/App/issues/5873

### Notes

The `background transparent` was needed to fix a weird behaviour on Android.

### Tests
1. Navigate to a workspace
2. Click on Reimburse receipts
3. Check the copy icon next to receipts@expensify.com
4. When taping the link, the `grey clipboard` icon should change to a `green checkmark`

### QA Steps
As above

### Tested On


### Screenshots

| Platform | Clipboard | Checkmark |
|:---:|:---:|:---:|
| Android | <img alt="ICON_ANDROID_A" src="https://user-images.githubusercontent.com/143615/143514383-2e55eadc-af72-4cb0-8ea7-d4906247c58f.png"/> | <img alt="ICON_ANDROID_B" src="https://user-images.githubusercontent.com/143615/143514381-bf445260-295f-43fe-9096-09d820402e5e.png"/> |
| Mobile Chrome | <img alt="ICON_MOBILE_CHROME_A" src="https://user-images.githubusercontent.com/143615/143514379-b9575852-d9a3-4406-b7eb-473e6b07b76f.png"/> | <img alt="ICON_MOBILE_CHROME_B" src="https://user-images.githubusercontent.com/143615/143514376-cd115770-9465-4d44-b850-c36234684f7d.png"/> |
| iPhone | <img alt="ICON_IOS_A" src="https://user-images.githubusercontent.com/143615/143514401-745907c7-adbc-4dfe-92e5-67876a396a1a.png"/> | <img alt="ICON_IOS_B" src="https://user-images.githubusercontent.com/143615/143514400-b7d8a175-dd8b-40ae-ace9-d4fbc234d955.png"/> |
| Mobile Safari | <img alt="ICON_MOBILE_SAFARI_A" src="https://user-images.githubusercontent.com/143615/143514397-884205a1-6377-4b31-80d9-3cb2fee8ec74.png"/> | <img alt="ICON_MOBILE_SAFARI_B" src="https://user-images.githubusercontent.com/143615/143514395-bf55ee42-3c8a-4c5a-90da-6696dc8dabd5.png"/> |
| Chrome | <img alt="ICON_CHROME_A" src="https://user-images.githubusercontent.com/143615/143514393-c03a7874-d7bb-4241-8a01-936917ee9670.png"> | <img alt="ICON_CHROME_B" src="https://user-images.githubusercontent.com/143615/143514391-bc97ed3b-57f9-43b9-8768-014c81a48e83.png"> |
| Safari | <img alt="ICON_SAFARI_A" src="https://user-images.githubusercontent.com/143615/143514390-6c718302-8cb2-4ee9-8fea-412dbd41234e.png"> | <img alt="ICON_SAFARI_B" src="https://user-images.githubusercontent.com/143615/143514388-e493406c-78bd-4c74-bd12-ebd8d4d277d1.png"> |
| Desktop | <img alt="ICON_DESKTOP_A" src="https://user-images.githubusercontent.com/143615/143514387-653ea772-d9ff-4581-a08e-065aa22713a8.png"> | <img alt="ICON_DESKTOP_B" src="https://user-images.githubusercontent.com/143615/143514384-e07f9c2a-3cd0-4a13-9af5-c54a6a78eb28.png"> |
